Pull request template (#140)
greenelab/covid19-review: .github/pull_request_template.md

<!-- Hi there! Please use the template below as a guide for opening a pull request.

> This template is designed to help the pull request process go smoother but if anything doesn't make sense to you, please open the pull request anyway and we will sort it out. Thank you for your contribution!! -->

Reviewer: ```suggestion
This template is designed to help the pull request process go smoother.
If anything doesn't make sense to you, please open the pull request anyway and leave entries blank as needed.
Thank you for your contribution!! -->
```

### Description of the proposed additions or changes
<!-- If the additions or changes aren't self explanatory feel free to leave a description or notes here. -->

### Related issues
<!-- GitHub will link issues to pull requests if you list the related issue numbers like "#2, #17". -->

> ### People tagged for review

Reviewer: ```suggestion
### Suggested reviewers (optional)
```

All the fields are optional, but this one seems even more optional than the others.  Is that worth noting?

> <!-- If there are particular people you think should review this pull request, please list their github handle like "@rando2".-->

Reviewer: ```suggestion
<!-- If there are particular people you think should review this pull request, please list their GitHub handle like "@rando2".-->
```

### Checklist
<!-- Please mark the following tasks as complete by putting an "x" in the box like "[x]".
If a particular item is not applicable to your pull request, please delete the item. -->

> - [ ] Text is formatted so that each sentence is on it's own line.

Reviewer: ```suggestion
- [ ] Text is formatted so that each sentence is on its own line.
```

<!-- see why this is recommended here:https://github.com/greenelab/covid19-review/blob/627a5f45a2d1fcb2150d1466df63bb848f22a3d4/USAGE.md#manubot-markdown -->

> - [ ] Pre-prints cited in this pull request have a github issue opened so that they can be reviewed.

Reviewer: ```suggestion
- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.
```

Have we been requiring an issue for every citation?  Or only the primary citations?
